modules/hal_nordic/nrfs/Kconfig

```diff
@@ -0,0 +1,126 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+config NRF_HAS_NRFS
+	bool
+
+config NRFS_HAS_CLOCK_SERVICE
+	bool
+
+config NRFS_HAS_CONST_LATENCY_SERVICE
+	bool
+
+config NRFS_HAS_DIAG_SERVICE
+	bool
+
+config NRFS_HAS_DVFS_SERVICE
+	bool
+
+config NRFS_HAS_MRAM_SERVICE
+	bool
+
+config NRFS_HAS_PMIC_SERVICE
+	bool
+
+config NRFS_HAS_RESET_SERVICE
+	bool
+
+config NRFS_HAS_TEMP_SERVICE
+	bool
+
+config NRFS_HAS_VBUS_DETECTOR_SERVICE
+	bool
+
+menu "nRF Services"
+	depends on NRF_HAS_NRFS
+
+config NRFS
+	bool "nRF Services Support"
+	select NRFS_LOCAL_DOMAIN if (SOC_NRF54H20_CPUAPP || SOC_NRF54H20_CPURAD)
+	depends on NRF_HAS_NRFS
+	help
+	  This option enables the nRF Services library
+
+config NRFS_LOCAL_DOMAIN
+	bool "nRF Services Local Domain Support"
+	select NRFS_BACKEND_IPC_SERVICE
+	help
+	  This option enables the nRF Services Local Domain libraries
+
+config NRFS_BACKEND_IPC_SERVICE
+	bool "IPC service backend for NRFS"
+	depends on NRFS
+	depends on $(dt_alias_enabled,ipc-to-cpusys)
+	select IPC_SERVICE
+	select IPC_SERVICE_BACKEND_ICMSG
```

Comment:
Is it needed? The backend should be automatically enabled based on the enabled node in dts: https://github.com/zephyrproject-rtos/zephyr/blob/a6400b42feeb2cc44aacb14d7b29370602cb9ce2/subsys/ipc/ipc_service/backends/Kconfig#L19-L23